pyopendds/dev/pyidl/__main__.py

```diff
@@ -72,45 +86,35 @@ def mk_tmp_package_proj(args: argparse.Namespace):
                               include_dirs=args.include_paths,
                               venv_path=os.environ['VIRTUAL_ENV']))
 
-    # Create setup.py
-    mk_tmp_file(f"{args.output_dir}/setup.py",
-                gen_setup(target_name=args.package_name))
-
-    # Create a the empty __init__.py to indicate the project is a package
-    # subprocess.run(['mkdir', args.package_name],
-    #                cwd=args.output_dir)
-    # mk_tmp_file(f"{args.output_dir}/{args.package_name}/__init__.py", "")
-    #
-    # # Install a dummy python package [package_name]
-    # print(f"Init dummy '{args.package_name}' Python package...")
-    # subprocess.run(['python3', 'setup.py', 'install'],
-    #                cwd=args.output_dir)
-    
     # add args to the make command
-    if args.make_opts=="":
-    	make_opts = []
+    if args.make_opts == "":
+        make_opts = []
     else:
-    	make_opts = args.make_opts.split(" ")
-
+        make_opts = args.make_opts.split(" ")
 
     # Run cmake to prepare the python to cpp bindings
-    subprocess.run(['mkdir', 'build'], cwd=args.output_dir)
-    subprocess.run(['cmake', '..'], cwd=f"{args.output_dir}/build")
-    subprocess.run(['make'] + make_opts, cwd=f"{args.output_dir}/build")
+    subprocess.run(['mkdir', '-p', 'build'], cwd=args.output_dir)
+    subprocess_check_run(['cmake', '..'], cwd=f"{args.output_dir}/build")
+    subprocess_check_run(['make'] + make_opts, cwd=f"{args.output_dir}/build")
```

Comment:
Could you add an option -jX that can be passed to pyidl as a parameter on this line so that we can use multithreading for python packages compilation ?

Reply:
Do you mean to add the parameter hardcoded by default ?

Because it seems that @PaulRousse already added this feature. We can call `pyidl file.idl -m "-j X"` option or whatever make option is passed in a string that follows `-m` (just pay attention to spacing as the string must be parsed to a list).

In the code above, it is the `make_opts` variable that is passed to `make`

Reply:
Yes, I did this thinking that we might forward other arguments than -jX to the make command. But it is not really handy and 
 I only use "-jX". If you modify it to **pyidl -jX ...** instead of  **pyidl -m "-jX" ...**, it is OK for me.

Reply:
The -m "-jX" is fine for me. I thought that this was a local fix that Paul had made and hadn't checked that it had been mergeD.
OK for me.